juju/juju
# feat: update description package

Updates the description package, brings in the new charm actions, config and LXD profile. This allows us to create the application without the current work around.

Doing so means that will have all the data that we expect to keep RI.

Work to correctly insert the charm will come as subsequent changes.





## Checklist




## QA steps

```sh
$ juju bootstrap lxd src40
$ juju add-model moveme
$ juju deploy ubuntu

$ juju bootstrap lxd dst40

$ juju migrate src40:moveme dst40
```

Ensure we can do the same with 3.6 into main

```sh
$ juju bootstrap lxd src36
$ juju add-model moveme
$ juju deploy ubuntu

$ juju bootstrap lxd dst40

$ juju migrate src36:moveme dst40
```


## Links



**Launchpad bug:** https://bugs.launchpad.net/juju/+bug/

**Jira card:** [JUJU-6411](https://warthogs.atlassian.net/browse/JUJU-6411)



[JUJU-6411]: https://warthogs.atlassian.net/browse/JUJU-6411?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
